### PR TITLE
Instructions for $TMPDIR on login nodes and general about login nodes

### PR DESCRIPTION
**Pull request by Nortamo**
**Do not merge yet!**
I will look at this again tomorrow.

### CODE REVIEW

File: docs/computing/disk.md [hunk 1]
```diff
@@ -126,3 +126,20 @@ quota regardless of how much data you actually have in the scratch
...
+## Additional disk areas
+
+### Login nodes
+
+All of the login nodes have 2900 GiB of fast local storage. The storage
+is located under `$TMPDIR` and is separate for each login node.  
+
+The local storage is good for compiling applications and performing 
+pre- and postprocessing that require heavy IO operations, for example packing and unpacking 
+archive files. Remember to move your data to a shared disk area afterwards.
```
Comment (galfthan) [inline docs/computing/disk.md]: is there any cleaning of this data?

File: docs/computing/system.md [hunk 2]
```diff
@@ -40,12 +40,19 @@ A detailed list of the nodes:
...
+    The login nodes can be used for **light** pre- and postprocessing, compiling
+    applications and moving data. All other tasks are to be done using the 
+    [batch job system](running/getting-started.md). Programs not adhering to these
+    rules will be terminated without warning.
 
 
 ### Storage
 
-Puhti has a 5+ PB Lustre parallel storage system providing space for home, project and scratch storages. 
+Puhti has a 5+ PB Lustre parallel storage system providing space for [home](disk.md#home-directory),
```
Comment (galfthan) [inline docs/computing/system.md]: We could fix at the same time. It is actually 4.8 PB...

**Comment by galfthan** [comment]
Looks pretty good already now.

File: docs/computing/disk.md (hunk 1 above)
Comment (Nortamo) [reply docs/computing/disk.md]: Clean up if files have not been accessed in 10 days
`/usr/sbin/tmpwatch "$flags" -x /local_scratch/build 10d /local_scratch`
Should we mention it explicitly or just say that files are frequently cleaned?

File: docs/computing/system.md (hunk 2 above)
Comment (Nortamo) [reply docs/computing/system.md]: I will change to the correct number

**Pull request merged**